### PR TITLE
10.2.0 Changelog

### PR DESCRIPTION
**Pull request by tbiethman**

### CODE REVIEW

File: content/_changelogs/10.2.0.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## 10.2.0
+
+_Released 6/21/2022_
+
+**Features:**
+
+- Cypress is now available natively on Apple Silicon/M1. Addresses
+  [#19908](https://github.com/cypress-io/cypress/issues/19908).
+- Cypress is now available natively on arm64/aarch64 systems running Linux.
```
Comment (emilyrohrbough) [inline content/_changelogs/10.2.0.md:9]:
```suggestion
- Cypress is now available natively on ARM64 or AArch64 systems running Linux.
```

File: content/_changelogs/10.2.0.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- Suppressed a misleading Chromium warning about CertVerifyProcBuiltin that
+  would appear when visiting HTTPS sites in Cypress. Fixed
+  [#22128](https://github.com/cypress-io/cypress/discussions/22128).
+- Users will be able to see the custom reason-phrase in the HTTP response if the
+  custom reason-phrase is set. Otherwise, a default status message is returned
+  based on the corresponding status code as earlier. Fixed
+  [#16973](https://github.com/cypress-io/cypress/issues/16973).
+- Fixed an issue where Cypress would crash with an EMFILE error if the process
+  ran into the file descriptor limit. Cypress will now retry file reads using
+  graceful-fs. Fixed
```
Comment (emilyrohrbough) [inline content/_changelogs/10.2.0.md:26]: I'd either drop mention of graceful-fs or we should expand on why this is matters. We don't typically reference dependencies in the changelog like this

Comment (tbiethman) [reply content/_changelogs/10.2.0.md]: I removed the reference to graceful-fs.

File: content/_changelogs/10.2.0.md (hunk 1 above)
Comment (tbiethman) [reply content/_changelogs/10.2.0.md]: Updated, I used `and` rather than `or`

**Comment by tbiethman** [comment]
Note that one more entry was added for a commit from this morning: https://github.com/cypress-io/cypress-documentation/pull/4582/commits/dc3c1ffac1f37e96fa9c67f4aaf528b03a82d36d

**Review by emilyrohrbough — APPROVED** [review]

**Comment by tbiethman** [comment]
Merging this into release branch, it will get more eyes on it 👀 prior to merging

**Pull request merged**